bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js

```diff
@@ -117,6 +117,7 @@ export default function addMeeting(meeting) {
     systemProps: {
       html5InstanceId: Number,
     },
+    groups: Array,
```

Comment:
Note for myself - when this change is merged, everyone would need to use 2.5 akka-apps packages (or re-compile their akka-apps locally)